### DISCUSSION TITLE
How to push from another account in VSCode after getting suspended from the main account.

### DISCUSSION THREAD

**Discussion by Rizz-33** [community/community#110959] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

```
$ git push
remote: Your account is suspended. Please visit https://support.github.com for more information.
fatal: unable to access 'https://github.com/Rizz-33/Taco-Loco.git/': The requested URL returned error: 403
```
I'm getting this error.

**Comment by mrsahuashish** [comment]
If you have been suspended from your main GitHub (or any other Git service) account and need to push code to a repository using a different account in Visual Studio Code (VSCode), you'll need to update your Git configuration to use the credentials of the alternate account. This involves changing both the Git global/user configuration settings and potentially the credential storage depending on your setup.

git config --global user.name "YourAlternateUsername"
git config --global user.email "your.alternate@example.com"

**Reply by Rizz-33** [reply]
I used this method and tried to push changes. But I'm getting the below error.

```
$ git push
fatal: The current branch main has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin main

To have this happen automatically for branches without a tracking
upstream, see 'push.autoSetupRemote' in 'git help config'.
```
Even if I run the command `git push --set-upstream origin main` , I'm getting the error mentioned in the question.
